src/pip/_internal/utils/deprecation.py

```diff
@@ -11,6 +11,15 @@
 from pip import __version__ as current_version
 
 DEPRECATION_MSG_PREFIX = "DEPRECATION: "
```

Comment:
@uranusjr thanks for the quick review!

I had deleted this variable, but since it's used [here](https://github.com/pypa/pip/blob/a53f8888bfc335677237bd971437c556670ed726/src/pip/_internal/utils/logging.py#L116) I added it back in my last commit https://github.com/pypa/pip/pull/10167/commits/8df3def385ef909233b9539fedd99b4ec3a20478. Removing it would involve reworking the `get_message_start()` function, so it seems easiest to leave it in for now.

Reply:
OK, thanks for checking.